src/Nri/Ui/Table/V8.elm

```diff
@@ -0,0 +1,368 @@
+module Nri.Ui.Table.V8 exposing
+    ( Column, SortDirection(..), custom, string, rowHeader, placeholderColumn
+    , view, viewWithoutHeader
+    , viewLoading, viewLoadingWithoutHeader
+    )
+
+{-| Upgrading from V7:
+
+  - New attribute added to define `alternatingRowColors`, set it to true to get default behavior.
+  - Consider moving to `SortableTable`, as in V4 a non-interactive table renders
+    just the same but has additional flexibility in the API and will make future
+    upgrades easier.
+
+@docs Column, SortDirection, custom, string, rowHeader, placeholderColumn
+
+@docs view, viewWithoutHeader
+
+@docs viewLoading, viewLoadingWithoutHeader
+
+-}
+
+import Accessibility.Styled.Style as Style
+import Css exposing (..)
+import Css.Animations
+import Css.Global
+import Html.Styled as Html exposing (..)
+import Html.Styled.Attributes as Attributes exposing (css)
+import Nri.Ui.Colors.V1 exposing (..)
+import Nri.Ui.Fonts.V1 exposing (baseFont)
+
+
+{-| Closed representation of how to render the header and cells of a column
+in the table
+-}
+type Column data msg
+    = Column (Html msg) (data -> Html msg) Style (data -> List Style) (Maybe SortDirection) CellType Bool
```

Comment:
All other attributes don't make much sense when representing a placeholder column. Does that mean we should have two separate constructors?

Reply:
Make sense. I will add a new constructor for that.